src/common/const.ts

```diff
@@ -198,6 +199,19 @@ export const DOMAINS_HIDE_MORE_INFO = [
   "select",
 ];
 
+/** Domains that render a special element instead of a text value when rendered in a row.
+ *  Those rows should then not show a cursor pointer when hovered (which would normally
+ *  be the default) unless the element itself enforces it (e.g. a button). Also those elements
+ *  should not act as a click target to open the more info dialog (the row name and state icon
+ *  still do of course) as the click might instead e.g. activate the input field that this row shows.
+ */
+export const DOMAINS_NO_POINTER_CLICK_IN_ROW = [
```

Comment:
That was my initial name as well, but that might lead people to e.g. add `input_boolean` since it's an input, but doesn't need to be in here since the toggle button that it renders already takes care of the cursor and the action handlers.

Reply:
But it also wouldnt hurt would it?

Reply:
Would probably still work if we included them, but result in unnecessary extra `<div>` and event listeners.

Reply:
I had it reversed in my head, but actually all domains that show an input element should be included in that const to not create extra `<div>` elements that are not necessary.